libs/designer/src/lib/ui/panel/nodeDetailsPanel/tabs/monitoringTab/footerPanel.tsx

```diff
@@ -0,0 +1,56 @@
+import constants from '../../../../../common/constants';
+import { useActionMetadata, useRunInstance } from '../../../../../core/state/workflow/workflowSelectors';
+import type { BoundParameters } from '@microsoft/logic-apps-shared';
+import { HostService, isNullOrEmpty, getPropertyValue, equals } from '@microsoft/logic-apps-shared';
+import { ValueLink } from '@microsoft/designer-ui';
+import { useCallback, useMemo } from 'react';
+import { useIntl } from 'react-intl';
+
+interface FooterPanelProps {
+  selectedNodeId: string;
+  outputs: BoundParameters;
+}
+
+export const FooterPanel: React.FC<FooterPanelProps> = (props) => {
+  const { selectedNodeId, outputs } = props;
+  const actionMetadata = useActionMetadata(selectedNodeId);
+
+  const nodeType = actionMetadata?.type ?? '';
+  const runInstance = useRunInstance();
+  const intl = useIntl();
+
+  const intlText = {
+    showLogicAppRun: intl.formatMessage({
+      defaultMessage: 'Show Logic App run details',
+      id: 'y6aoMi',
+      description: 'Show Logic App run details text',
+    }),
+  };
+
+  const getChildRunNameFromOutputs = (outputs: any): string | undefined => {
+    if (!isNullOrEmpty(outputs)) {
+      const headers = getPropertyValue(outputs, 'headers');
+      if (headers && headers.value) {
+        return getPropertyValue(headers.value, 'x-ms-workflow-run-id');
+      }
```

Comment:
you can just use getObjectProperyValue(outputs, ['headers', 'value', 'x-ms-workflow-run-id'])